src/pyscipopt/scip.pxi

```diff
@@ -2834,6 +2834,35 @@ cdef class Model:
         PY_SCIP_CALL(SCIPreleaseCons(self._scip, &scip_cons))
 
         return pyCons
+    
+    def addPiecewiseLinearCons(self, X, Y, a, b):
+        """add piecewise relation with multiple selection formulation
```

Comment:
Maybe add a reference here or more explanation. Also mention somewhere that it is 2d.